src/gluonts/distribution/transformed_distribution.py

```diff
@@ -77,6 +77,13 @@ def event_shape(self) -> Tuple:
         assert isinstance(self._event_shape, tuple)
         return self._event_shape
 
+    @property
+    def mean(self) -> Tensor:
+        s = self.base_distribution.mean
+        for t in self.transforms:
+            s = t.f(s)
+        return s
```

Comment:
Is this correct? I read this as "mean(f(X)) = f(mean(X))", which is the case if f is linear, but not necessarily otherwise

Reply:
Oups, thanks for the catch! I removed it.